pkg/api/types.go

```diff
@@ -1566,6 +1566,11 @@ type PodSpec struct {
 	// If specified, the fully qualified Pod hostname will be "<hostname>.<subdomain>.<pod namespace>.svc.<cluster domain>".
 	// If not specified, the pod will not have a domainname at all.
 	Subdomain string `json:"subdomain,omitempty"`
+	//Optional: Docker "--shm-size" support. Defines the size of /dev/shm in the IPC namespace of the pod.
```

Comment:
Excuse me,, I am not sure I understand the point here. The planned functionality was: if the user defines ShmSize in the pod manifest, the defined size is sent to the container runtime. If ShmSize is not defined in the pod manifest, no size definition is sent to the container runtime, and the container runtime applies its default value.
In this case the container runtime is Docker, that is true. Though, I do not understand this one "avoid docker specific defaulting behaviors in the API ". Where did we apply Docker specific defaulting behavior on the API? ShmSize is optional, is that the problem?